test-packages/e2e-tests/rest.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const OpenAPIBackend = require('openapi-backend').default;
```

Comment:
cool stuff. That you can create a backend from spec.

Reply:
Yeah, but it is less cool than it looks like on first sight. The data still needs to be provided manually

Reply:
ehhh